Gemfile

```diff
@@ -26,13 +26,8 @@ gem 'prometheus_exporter', '2.1.0'
 gem 'puma', '>= 5.6.4'
 gem 'rails', '~> 7.0.8'
 gem 'redis', '~> 5.1.0'
-<<<<<<< HEAD
-gem 'sentry-rails', '~> 5.17.1'
 gem 'sentry-sidekiq', '~> 5.17.3'
-=======
 gem 'sentry-rails', '~> 5.17.3'
```

Comment:
I do not know how Rubocop is configured here but I believe that by default it wants gems in Gemfile to be in alphabetical order.

Also, this commit can be merged into the previous as it was in that one that this merge conflict appeared.